src/graphql/operations/votes.ts

```diff
@@ -55,15 +55,13 @@ async function query(parent, args, context?, info?) {
   if (requestedFields.space && votes.length > 0) {
     const spaceIds = votes.map(vote => vote.space.id).filter((v, i, a) => a.indexOf(v) === i);
     const query = `
-      SELECT id, settings FROM spaces
+      SELECT * FROM spaces
```

Comment:
Missing the `, spaces.flagged as spaceFlagged, spaces.verified as spaceVerified` at line 79 below